services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -476,7 +476,7 @@ public VeniceControllerConfig(VeniceProperties props) {
     this.parentExternalSupersetSchemaGenerationEnabled =
         props.getBoolean(CONTROLLER_PARENT_EXTERNAL_SUPERSET_SCHEMA_GENERATION_ENABLED, false);
     this.systemSchemaInitializationAtStartTimeEnabled =
-        props.getBoolean(SYSTEM_SCHEMA_INITIALIZATION_AT_START_TIME_ENABLED, true);
+        props.getBoolean(SYSTEM_SCHEMA_INITIALIZATION_AT_START_TIME_ENABLED, false);
```

Comment:
Why this change?

Reply:
I made a change in this commit. When this config is enabled but neither url nor d2 config is provided, throw exception instead of aborting the initialization and continuing that startup of controller.
```
throw new VeniceException(
            "System schema initialization is enabled but neither controller url nor d2 config is provided.");
```
If the default of this config continues to be true, open source users who do not provide controller url or d2 will fail to start controllers. So I changed it to false.

Reply:
I see.
Let us make sure we have this config enabled in LinkedIn env.